Src/CSharpier/SyntaxPrinter/Token.cs

```diff
@@ -113,27 +114,34 @@ public static Doc PrintLeadingTriviaWithNewLines(SyntaxTriviaList leadingTrivia)
     private static Doc PrivatePrintLeadingTrivia(
         SyntaxTriviaList leadingTrivia,
         bool includeInitialNewLines = false,
-        bool skipLastHardline = false
+        bool skipLastHardline = false,
+        bool useLastWasLine = false
```

Comment:
This is kind of a terrible name. What does it use `lastWasLine` for?
To skip consecutive lines? If so, name it as such.

Reply:
I realized it was basically the equivalent of !printNewLines and figured out how to get rid of it completely. Even with a better name the code was still ugly.